fixtures/sponsors.json

```diff
@@ -0,0 +1,23 @@
+[
+{
+    "fields": {
+        "added": "2016-04-11T21:55:00Z",
+        "name": "Continuum Analytics",
+        "display_url": "",
+        "sponsor_logo": 4,
```

Comment:
`sponsor_logo` does not exist in the database.
I had to comment out this line and add a file through the admin interface.